tmt/steps/prepare/install.py

```diff
@@ -48,7 +48,7 @@ def prepare_packages(self) -> None:
 
         # Detect local, debuginfo and repository packages
         for package in self.packages:
-            if package.endswith('.rpm'):
+            if package.endswith('.rpm') and not re.match(r'^http(s)?://', package):
```

Comment:
* this way, the package is going to land in `self.repository_packages` (thanks to final `else`), which does not seem right. Maybe we need another list, "remote_packages"?
* also, the package wouldn't be checked for being debuginfo/debugsource, unlike the local packages - could this cause issues? The comment after `elif` suggests this might be a problem. @psss could know more, according to `git log` :)

Reply:
Yes, package starting with `https://` should not be regarded as `repository_packages`. I have added an another list - `self.remote_packages` and method `install_from_url()` to handle it. @happz, please help to review again, thanks!

Reply:
Yes, I'd agree that it makes sense to have a dedicated list for remote packages and special method for their installation as some of the implementation such as `rpm-ostree` might not support fetching remote packages. What about filtering out the remote packages as the very first condition?

Regarding `debuginfo` packages: Currently we ignore them among local packages as well, so we could state that those are supported from repositories only, or fix it for both. Probably not a high priority now.

Anyway, it would be good to document this new change in the [specification](https://tmt.readthedocs.io/en/stable/spec/plans.html#install) and add at least a basic test coverage.

Reply:
> What about filtering out the remote packages as the very first condition?

DONE.

> Regarding debuginfo packages: Currently we ignore them among local packages as well, so we could state that those are supported from repositories only, or fix it for both. Probably not a high priority now.

Fixed.

Reply:
> Anyway, it would be good to document this new change in the [specification](https://tmt.readthedocs.io/en/stable/spec/plans.html#install) and add at least a basic test coverage.

Hi @psss, the related document file `spec/plans/prepare.fmf` is updated, please help to review, thanks!

Reply:
Thanks for addressing all comments.